src/app/compiler.js

```diff
@@ -153,10 +155,11 @@ function Compiler (handleImportCall) {
       compilationFinished({error: 'Compiler not yet loaded.'})
     })
 
-    var newScript = document.createElement('script')
```

Comment:
I remember there might have been some magic involved and I specifically did not use jquery here, but I might also be mistaken.